### PR TITLE
Fixes in() filter (non-macro syntax) and adds unit tests

### PR DESCRIPTION
**Pull request by vertexmc**

### CODE REVIEW

File: synapse/lib/storm.py [hunk 1]
```diff
@@ -1331,11 +1332,6 @@ def _stormOperSetProp(self, query, oper):
             if props:
                 [core.setTufoProps(node, **props) for node in nodes]
 
-    def _iterPropTags(self, props, tags):
```
Comment (vertexmc) [inline synapse/lib/storm.py]: This was unused

File: synapse/lib/storm.py [hunk 2]
```diff
@@ -749,29 +750,6 @@ def cmpr(tufo):
 
         return cmpr
 
-    def _stormOperShowCols(self, query, oper):
```
Comment (vertexmc) [inline synapse/lib/storm.py]: I just moved these down with the rest of the opers

File: synapse/lib/storm.py [hunk 3]
```diff
@@ -713,19 +713,20 @@ def _reqOperArg(self, oper, name):
...
             return tufo[1].get(prop) is not None
 
         return cmpr
 
     def _cmprCtorIn(self, oper):
-        prop = self._reqOperArg(oper, 'prop')
-        valus = self._reqOperArg(oper, 'valu')
-        if len(valus) > 12: # TODO opt value?
-            valus = set(valus)
+        prop, valu = oper[1].get('args')
```
Comment (vertexmc) [inline synapse/lib/storm.py]: Like the `range` filter, it was not possible to get `prop` and `valus` into this fn.

File: synapse/lib/storm.py (hunk 1 above)
Comment (vEpiphyte) [reply synapse/lib/storm.py]: can't find it anywhere else in the codebase

Comment (vEpiphyte) [reply synapse/lib/storm.py]: was support code for _stormOperAllTag, no longer used (see 5ecbf19f14fbc8fe8a53718130d139a286e4ad10)

**Comment by vEpiphyte** [comment]
codecov diff % is misleading because we moved previously untested code

**Pull request merged**